pkg/converter/tool/builder.go

```diff
@@ -117,6 +128,46 @@ func Pack(option PackOption) error {
 	return nil
 }
 
+func packRef(option PackOption) error {
+	args := []string{
+		"create",
+		"--log-level",
+		"warn",
+		"--type",
+		"targz-ref",
+		"--blob-inline-meta",
+		"--features",
+		"blob_toc",
+		"--blob",
+		option.BlobPath,
+	}
+	args = append(args, option.SourcePath)
+
+	ctx := context.Background()
+	var cancel context.CancelFunc
+	if option.Timeout != nil {
+		ctx, cancel = context.WithTimeout(ctx, *option.Timeout)
+		defer cancel()
+	}
+
+	logrus.Debugf("\tCommand: %s %s", option.BuilderPath, strings.Join(args, " "))
```

Comment:
Should avoid print messages in a library package?

Reply:
It's ok for printing debug messages.